transmit/transmit.go

```diff
@@ -160,7 +160,7 @@ func (d *DefaultTransmission) processResponses(
 				if r.Err != nil {
 					log = log.WithField("error", r.Err.Error())
 				}
-				log.Logf("non-20x response when sending event")
+				log.Logf("error when sending event")
```

Comment:
Would `"error sending event or processing the response"` be more helpful to avoid narrowing in the mind of the reader the scope of where the error popped up?

Reply:
Mentioning response processing feels like leaking libhoney internals into here, but maybe there's a different way to say it that encapsulates sending/processing? 

Error transmitting event?
Event transmission error?

Also, with the additional context in libhoney, it will say something like:
`ERROR: error when sending event: got OK HTTP response, but couldn't read response body`

Reply:
Works for me then!